### DISCUSSION TITLE
SCRUM PREP MILESTONE 1-3

### DISCUSSION THREAD

**Discussion by Keiven-Soto** [uprm-inso4116-2024-2025-s1/semester-project-multiplayer-dominoes#115] [General]
This week, i worked on revising and closing pull request. Generated metric scores and met with the other manager to discuss the future outlook of the project where we talked about the expectations for the next milestone and the project presentation.

**Comment by ianramospr** [comment]
This is week I was working on the Game overlays for when a player wins, looses or it's their turn to play.

**Comment by Saul-Caban-UPRM** [comment]
Worked on finishing the achievement assets for the task #66 and close the task . Discussed about future task for milestone 2 and milestone 3 with Ui Team leader.

**Comment by ebrain-mendoza121** [comment]
This week I fixed the ui of the Lobby page to work in accord with the gamestate and the login. Completed the validation and verification documentation task.

**Comment by PinkSylvie** [comment]
By this week I had finished my tasks for milestone 1 aside my team leader tasks, so I’ve been keeping track of my team’s progress towards the end of the Milestone and created my tasks for the next one.

**Comment by JadiellMV** [comment]
This week I implemented the authentication endpoints to the backend, and also filled the user's table with some dummy data to be able to work with some information in the meantime.

**Comment by AngelCIICMorales** [comment]
This week I organized two meetings. I also reviewed several issues and I am finalizing a lecture topic task.

**Comment by Joaquin939** [comment]
This week I completed the rules button wireframe and it's implementation.

**Comment by sandy-marrero** [comment]
- Completed and added to the documentation #44, #47, #46. 
- Completed the issue #21   and made a PR #99 which has been completed.
- Assisted to team meetings.
- Did some reviews on a PR.
- Made PR #101 to align project favicon, icon with current logo.

**Comment by AnthonyMed13** [comment]
Completed remaining issues: concept analysis #26 , other activities than just developing source code #22 , and Implement basic user registration endpoints #18

**Comment by ilikecheese117** [comment]
Merged save game system into main and created test files for it.

**Comment by Adriana-Isabel** [comment]
I attended the weekly team meeting and brainstormed ideas for the next milestones.

For Milestone 2:
> Implement the Header to the Webapp
> Implement the Footer to the Webapp

**Comment by RicardoJRSs** [comment]
This week I finished all my tasks and added documentation

**Comment by DavidCuevas3** [comment]
I finished working on the leaderbord wireframe and started preparing tasks for milestone 2

**Comment by FrancesC8** [comment]
This week I deployed the backend server in Vercel and the database in MongoDB Atlas. The backend server can be accessed here [https://multi-dominoes.vercel.app/](https://multi-dominoes.vercel.app/). Additionally, I completed the remaining documentation task for this milestone.

**Comment by Gaby-Concepcion** [comment]
For this final week of milestone , I worked on finalizing the user stories and epics, as well as finishing my asigned game state issues.

**Comment by frances-sepulveda** [comment]
This week I worked with @rukitoyordan to do the final details for the avatars and boards. Additionally, I worked on the documentation.

**Comment by Negroni1** [comment]
Close #64 Such that upon creating a user, a UUID is generated in the DB for the user

**Comment by Jvstinoo** [comment]
This week I completed tasks to add the pause button to the game as well as unit testing for general game logic.

**Comment by rukitoyordan** [comment]
For this week, I worked on powerup creation for the avatars, added 2 new board variations to the files, as well as handled current situation (1.2.1) on the documentation.

**Comment by nsfrontera** [comment]
This week i finished all my task, making the Rule engine and Achievements. They have been successfully merged to the main branch

**Comment by print-Eruki** [comment]
Finishing the ERD and high level architecture doagram to send to review asap

**Comment by G4vp** [comment]
This week I focused to review pull requests and help developers to fix merge conflicts and other problems when pushing code.

**Comment by fernandopizarro1** [comment]
This week I helped in the implementation of the tasks in relation to gameplay and AI by reviewing pull requests and giving them suggestions that upgrade their implementations.
